go.mod

```diff
@@ -50,6 +50,7 @@ require (
 )
 
 require (
+	github.com/aws/aws-sdk-go v1.44.312
```

Comment:
Should we go ahead and use `aws-sdk-go-v2`?

Reply:
Probably doesn't matter much either way, since the struct is only used to hold values. But why not, if its also available in v2: https://github.com/aws/aws-sdk-go-v2/blob/main/aws/credentials.go#L74-L93

I'll put out a revision shortly